hummingbot/client/ui/style.py

```diff
@@ -136,4 +138,6 @@ def hex_to_ansi(color_hex):
     "primary":                    "#ansigreen",  # noqa: E241
     "warning":                    "#ansibrightyellow",  # noqa: E241
     "error":                      "#ansired",  # noqa: E241
+    "tab_button.focused":         "bg:#ansigreen #ansiblack",  # noqa: E241
```

Comment:
I don't think this belongs here. hex_to_ansi() is just for converting individual color values from hex to windows ANSI codes because Windows command line is very limited. Styles of specific components are defined on the bottom of this file.